pkg/known/constant.go

```diff
@@ -53,6 +53,7 @@ const (
 // These are internal finalizer values to Clusternet, must be qualified name.
 const (
 	AppFinalizer            string = "apps.clusternet.io/finalizer"
+	SeFinalizer             string = "se.clusternet.io/finalizer"
```

Comment:
shall we use `AppFinalizer` instead?

or `apps.clusternet.io/serviceexports-finalizer`

Reply:
ok, AppFinalizer.